data/json/recipes/recipe_others.json

```diff
@@ -932,6 +932,23 @@
   ]
 },{
   "type" : "recipe",
+  "result": "crude_firestarter",
+  "category": "CC_ELECTRONIC",
+  "subcategory": "CSC_ELECTRONIC_TOOLS",
+  "skill_used": "electronics",
+  "difficulty": 0,
+  "time": 5000,
+  "reversible": true,
+  "autolearn": true,
+  "qualities":[
+    {"id":"HAMMER","level":2,"amount":1}
```

Comment:
If you reduce this to ````{"id":"HAMMER","level":2,"amount":1}```` it will be constructable with improvised (like a rock) means.